mkdocs/mkdocs.yml

```diff
@@ -60,6 +60,8 @@ nav:
         - 'api/phase_api.md'
         - 'api/trajectory_api.md'
         - 'api/transcriptions_api.md'
+    - 'Dymos Command Line':
```

Comment:
minor nitpick, but I'd call this section "Dymos Command Line Interface"

Reply:
I made this change, but the length makes the header on the left look a little weird.

Reply:
Hmm...maybe Dymos is redundant and we can just go with "Command line interface"